docs/deployment/heroku.md

```diff
@@ -5,69 +5,138 @@ sidebar_label: Heroku
 description: How to deploy Backstage to Heroku
 ---
 
-Heroku is a Platform as a Service (PaaS) designed to handle application
-deployment in a hands-off way. Heroku supports container deployment of Docker
-images, a natural fit for Backstage.
+Heroku is a Platform as a Service (PaaS) designed to simplify application deployment.
 
-## Configuring the CLI
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/chap/backstage-button)
```

Comment:
If we can remove this that would be ideal, just leaves us open to something going unmaintained that can be easily missed.

Reply:
No problem, removed